Array_1_Problem_3.java

```diff
@@ -0,0 +1,135 @@
+//Time Complexity : O(n)
+//Space Complexity: O(n)
+//Run successfully on leetcode
+//Problem : Finding the optimal pattern
+
+//As i was absent in the lecture due to my finals, i tried to come up with my own solution
+//I'm using a linkedhashset to maintain the order and for checking if the element has occureced or not
```

Comment:
But, what if the follow up question is regarding duplicates?

Reply:
LinkedHashset is an over kill here. Sowmya is right